### PR TITLE
Add Write-Ups and POCs

### PR DESCRIPTION
**Pull request by dhaval17**
**https://github.com/dhaval17/awesome-writeups**

**The List contains the WriteUps and POCs of security vulnerabilities**

# By submitting this pull request I confirm I've read and complied with the below requirements.

Failure to properly do so will just result in the pull request being closed and everyone's time wasted. Please read it twice. Most people miss many things.

- I have read and understood the [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/contributing.md) and the [instructions for creating a list](https://github.com/sindresorhus/awesome/blob/master/create-list.md).
- This pull request has a descriptive title. For example, `Add Name of List`, not `Update readme.md` or `Add awesome list`.
- The entry in the Awesome list should include a short description about the list project/theme (not the list itself), and should be added at the bottom of the appropriate category.
- The list I added
	- **has been around for at least 30 days,**
	- is the result of hard work and is the best I could possibly produce,
	- is a non-generated Markdown file in a GitHub repo,
	- is not a duplicate,
	- includes a succinct description of the project/theme at the top ([example](https://github.com/willempienaar/awesome-quantified-self)),
	- includes a project logo/illustration (whenever possible) at the top right of the readme ([example](https://github.com/sindresorhus/awesome-electron)). (The logo should link to the project website or any relevant website. The logo should be high-DPI, so set it to maximum half the width of the original image.),
	- only has awesome items **with descriptions** *(Awesome lists are curations of the best, not of everything)*,
	- contains the [awesome badge](https://github.com/sindresorhus/awesome/blob/master/awesome.md#awesome-badge) on the right side of the list heading,
	- has a Table of Contents section **named `Contents`** as the first section,
	- has an [appropriate license](https://github.com/sindresorhus/awesome/blob/master/awesome.md#choose-an-appropriate-license) *(Meaning something like CC0, NOT a code licence like MIT, BSD, Apache, etc)* *(If you use a license badge, it should be SVG, not PNG)*,
	- has [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/awesome.md#include-contribution-guidelines),
	- doesn't include a Travis badge,
	- has been checked for inconsistent formatting, spelling, and grammar:
		- Each link description starts with an uppercase character and ends with a period.
		- Drop all the `A` / `An` prefixes in the descriptions.
		- Consistent naming. For example, `Node.js`, not `NodeJS` or `node.js`.

### CODE REVIEW

**Comment by dhaval17** [comment]
@sindresorhus Anything ?

**Comment by dhaval17** [comment]
@sindresorhus can anyone help me out here ?

**Comment by itsjtwright** [comment]
Reread the contributing page, you have many issues, including that every item on the list needs a desc

**Comment by sindresorhus** [comment]
What @itsjtwright said.

**Comment by davisonio** [comment]
@itsjtwright could you make some changes? :)

Also: what are POCs? This should be addressed in the list to be newbie friendly.

**Comment by legodude17** [comment]
I would suggest renaming it `awsome-security-writeups` if that is what it is about.

**Comment by dhaval17** [comment]
> I would suggest renaming it awsome-security-writeups
@legodude17 Will do, Thanks

> what are POCs?
@davisonio Short for `Proof of Concept` for a vulnerability

File: readme.md [hunk 1]
```diff
@@ -377,6 +377,7 @@ Check out my [blog](https://blog.sindresorhus.com) and follow me on [Twitter](ht
 - [Honeypots](https://github.com/paralax/awesome-honeypots) - Deception trap, designed to entice an attacker into attempting to compromise the information systems in an organization.
 - [Incident Response](https://github.com/meirwah/awesome-incident-response)
 - [Vehicle Security and Car Hacking](https://github.com/jaredmichaelsmith/awesome-vehicle-security)
+- [WriteUps And POCs](https://github.com/dhaval17/awesome-writeups)
```
Comment (sindresorhus) [inline readme.md]: Should be `Write-Ups and POCs`

And is missing a description:

> The entry in the Awesome list should include a short description about the list project/theme **(not the list itself)**

**Comment by sindresorhus** [comment]
> I would suggest renaming it awsome-security-writeups if that is what it is about.

You also need to update your readme title and the title here.

**Comment by sindresorhus** [comment]
There's a lot of stuff to comment on regarding the list formatting, but I have little time. Just look at other list and try to better adhere to the Awesome style.

Items should have descriptions. In this case it would be nice with a short summary of the write-up.

Don't use my email in https://github.com/dhaval17/awsome-security-writeups/blob/master/code-of-conduct.md

File: readme.md (hunk 1 above)
Comment (dhaval17) [inline readme.md]: @sindresorhus Above done

**Comment by dhaval17** [comment]
@sindresorhus 
- Title in the push request and README.md changed appropriately
- Description added to push request
- Email Removed
- Adding more information to the items

**Comment by dhaval17** [comment]
@sindresorhus My items are not one word, These are self explanatory sentences which should make sense to people will littlest idea of information security

Nonetheless, I will be adding description to the items as much as I can over a period of time

If that's fine by you, Please approve the push request

File: readme.md [hunk 2]
```diff
@@ -393,9 +393,11 @@
 - [Honeypots](https://github.com/paralax/awesome-honeypots) - Deception trap, designed to entice an attacker into attempting to compromise the information systems in an organization.
 - [Incident Response](https://github.com/meirwah/awesome-incident-response)
 - [Vehicle Security and Car Hacking](https://github.com/jaredmichaelsmith/awesome-vehicle-security)
+- [Security Write-Ups and POCs](https://github.com/dhaval17/awsome-security-write-ups-and-POCs) - Curated list of write-ups and POCs from the world of information security.
```
Comment (sindresorhus) [inline readme.md]: The description here is just repeating the title. Please read my comment again in https://github.com/sindresorhus/awesome/pull/882#discussion_r112399953 It should be about the theme, not your list.

File: readme.md [hunk 3]
```diff
@@ -393,9 +393,11 @@
...
 - [Web Security](https://github.com/qazbnm456/awesome-web-security) - Security of web apps & services.
 
 
+
```
Comment (sindresorhus) [inline readme.md]: Don't do unrelated changes.

**Comment by sindresorhus** [comment]
The Markdown headers are too small. Main header should be `#`, main sections `##`, etc.

Use unordered lists, not ordered.

**Comment by sindresorhus** [comment]
Closing for lack of response.

**Pull request closed without merging**